modules/react/skeleton/lib/parts/skeletonShape.tsx

```diff
@@ -1,25 +1,11 @@
 import * as React from 'react';
-import styled from '@emotion/styled';
-import canvas from '@workday/canvas-kit-react/tokens';
 
-export const BOTTOM_MARGIN = 16;
+import {createComponent, styled, StyledType} from '@workday/canvas-kit-react/common';
+import canvas from '@workday/canvas-kit-react/tokens';
 
-const Shape = styled('div')<{
-  width?: number | string;
-  height?: number | string;
-  borderRadius?: number | string;
-  backgroundColor?: string;
-}>(({width, height, borderRadius, backgroundColor}) => {
-  return {
-    width: width ? width : '100%',
-    height: height ? height : '100%',
-    borderRadius: borderRadius ? borderRadius : 0,
-    backgroundColor,
-    marginBottom: BOTTOM_MARGIN,
-  };
-});
+const BOTTOM_MARGIN = 16;
```

Comment:
Nit: Replace with `space.s` value.

 I know this wasn't your addition, but this variable is only used in one place and the value is a part of our space scale. So, I'd rather we use `space.s` here.
 
 ```tsx
 import { space } from "@workday/canvas-kit-react/tokens";
 ...
 
 marginBottom: space.s,
 ```

Reply:
Another option would be to use `Box` for `Shape`, and then you don't need to import tokens at all. Your call.